packages/weex-vue-framework/factory.js

```diff
@@ -5899,7 +5899,7 @@ function createPatchFunction (backend) {
   }
 
   function emptyNodeAt (elm) {
-    return new VNode(nodeOps.tagName(elm).toLowerCase(), {}, [], undefined, elm)
+    return (typeof nodeOps.tagName(elm) == "string") ? new VNode(nodeOps.tagName(elm).toLowerCase(), {}, [], undefined, elm) : new VNode("", {}, [], undefined, elm)
```

Comment:
could be rewritten as 
```suggestion
    return new VNode((typeof nodeOps.tagName(elm) == "string") ? nodeOps.tagName(elm).toLowerCase() : '', {}, [], undefined, elm)
```